clang/lib/Sema/SemaDecl.cpp

```diff
@@ -11799,6 +11792,12 @@ static bool CheckMultiVersionAdditionalDecl(
 static bool CheckMultiVersionFunction(Sema &S, FunctionDecl *NewFD,
                                       bool &Redeclaration, NamedDecl *&OldDecl,
                                       LookupResult &Previous) {
+  const TargetInfo &TI = S.getASTContext().getTargetInfo();
+
+  // Abort if FMV is disabled.
```

Comment:
"Abort"  might be a miss leading comment. 
```suggestion
  // Check if FMV is disabled.
```